app/src/serial-cmdhandler.cpp

```diff
@@ -54,6 +54,7 @@ void SerialCmdHandler::cmdMenu(void)
     err("                          id-RelayID number(0,1,2...) ");
     err("                          x-Off/On(0-1); r-reset; y-reset timeout value");
     err(" inf                  Print device information data");
+    err(" n,new_name           Change name of the USBMUX");

```

Comment:
new_name is not supprted, please remove it :) You set only: {"n", std::bind(&SerialCmdHandler::processCmdSetName, this)}